tests/tensor/test_math_opt.py

```diff
@@ -2806,14 +2798,14 @@ def test_local_erf_minus_one(self):
         x = vector()
 
         f = function([x], erf(x) - 1, mode=self.mode)
-        assert [n.op for n in f.maker.fgraph.toposort()] == [erfc, mul]
+        assert [n.op for n in f.maker.fgraph.toposort()] == [erfc, neg]
```

Comment:
The `mul(-1, x)` gets optimized to `neg(x)`, now that the test includes the specialization rewrite phase